README.md

```diff
@@ -44,3 +44,95 @@ Ensure that you have the correct path by looking for the file `Beyond-All-Reason
 6. If developing Chobby also clone the code into the `games` directory. Follow the guide in the [Chobby README](https://github.com/beyond-all-reason/BYAR-Chobby#developing-the-lobby).
 
 More on the `.sdd` directory to run raw LUA and the structure expected by Spring Engine is [documented here](https://springrts.com/wiki/Gamedev:Structure).
+
+### Testing
+
+#### Unit Testing
+
+##### Install Lua 5.1 and LuaRocks
+
+*debian/linux*
+
+```zsh
+sudo apt install -y lua5.1 luarocks
+```
+*windows* (MSYS2, use UCRT64)
```

Comment:
I lack a Windows box to test this at the moment, unfortunately. Maybe someone else can chime in?